main.go

```diff
@@ -47,6 +50,13 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+const (
+	ControllerNamespaceEnvVar  = "NAMESPACE"
```

Comment:
Can you use a more specific name for the env variable?
`NAMESPACE` doesn't give the idea that you want the namespace of the modelmesh-controller.

Reply:
Same handling was already implemented in `ModelMesh-Controller`. In `ModelMesh-Controller` they are using `NAMESPACE` env to fetch controller namespace. I keep the same in `odh-model-controller` to maintain consistency.
https://github.com/opendatahub-io/modelmesh-serving/blob/dd0229fe52d6c90703eabeecfdee287dd3ac55fd/main.go#L69

Reply:
I think it depends on which controller you mean. If this is for "self", the naming in OK. But after reading the code, most likely, this should refer to the `modelmesh-controller` namespace rather than $self. If this is the case, the naming should reflect that.